python/tvm/topi/arm_cpu/mprofile/dsp/conv2d.py

```diff
@@ -120,7 +121,12 @@ def conv2d_nhwc_dsp_compute(cfg, data, kernel, strides, padding, dilation, out_d
         policy="factors",
         num_outputs=2,
         # TODO: check case with in_channels.value % 4 != 0 with AutoTVM
-        filter=None if cfg.is_fallback else lambda x: x.size[-1] % 4 == 0,
+        # temp fix: if non of the splits are divisible by 4, don't filter them
+        # to prevent an empty search space, and allow autotuning to select the best one
+        filter=None
+        if cfg.is_fallback
+        else lambda x: x.size[-1] % 4 == 0
```

Comment:
can you outline the lambda function into its own function here, if we wind up with this solution? generally speaking lambdas should be just 1-liners